packages/typespec-ts/test/integration/generated/scalar/src/clientDefinitions.ts

```diff
@@ -8,6 +8,16 @@ import {
   BooleanModelPutParameters,
   UnknownGetParameters,
   UnknownPutParameters,
+  DecimalTypeResponseBodyParameters,
+  DecimalTypeRequestBodyParameters,
```

Comment:
these cases should be supported in https://github.com/Azure/autorest.typescript/issues/2157